addons/auth_signup/res_users.py

```diff
@@ -194,7 +195,8 @@ def signup(self, cr, uid, values, token=None, context=None):
             if partner_user:
                 # user exists, modify it according to values
                 values.pop('login', None)
-                values.pop('name', None)
+                if single_email_re.match(partner_user.name) == None:
```

Comment:
From dashboard we are inviting user just by email and name and login are required so i have to put email in both fields. So when user sign up he can change his name (only if name is email id).  
![download 7](https://cloud.githubusercontent.com/assets/7629089/9111889/a07b7902-3c68-11e5-98b0-045aead96182.png)


![download 5](https://cloud.githubusercontent.com/assets/7629089/9111720/4a019954-3c67-11e5-8783-5fb4121476cb.png)